lib/bootloader_setup.pm

```diff
@@ -31,6 +32,19 @@ our @EXPORT = qw(
   tianocore_select_bootloader
 );
 
+sub boot_local_disk {
+    if (check_var('ARCH', 'ppc64le')) {
+        # TODO use bootindex to properly boot from disk when first in boot order is cd-rom
+        wait_screen_change { send_key 'ret' };
+        if (check_screen 'inst-slof', 10) {
```

Comment:
can you use `assert_screen [qw(inst-slow bootmenu)];` and "match_has_tag" instead of check_screen with non-zero timeout?